chain/client/src/rocksdb_metrics.rs

```diff
@@ -35,8 +35,7 @@ pub(crate) struct RocksDBMetrics {
 
 /// Returns column's name as name in the DBCol enum without '_' prefix for deprecated columns.
 fn col_verbose_name(col: near_store::DBCol) -> &'static str {
-    let name: &str = col.into();
-    name.strip_prefix("_").unwrap_or(name)
+    <&str>::from(col)
 }
```

Comment:
this fn probably doesn't pull its weight anymore?

Reply:
Though, I'd maybe add this as a convenience API to use instead of `<str>::from`

```rust
impl DBCol {
   pub fn name(&self) -> &'static str { self.into() }
}
```

Reply:
We used to have this function but then it turns out it was only ever used when Display could be used so I got rid of it.  At the moment it would be used in just two places so dunno really if it’s worth it.